### ISSUE TITLE
Application kicks me out of an account when I open streams

### ISSUE THREAD

**Issue by p3rs0n4l1ty** [nopbreak/TwitchMod#38]
As soon as I try to watch any stream a pop-up notification appears that states about connectivity issue. After that I try to open app settings and then a window appears which asks me to login into my account again.
![Screenshot_20210123_193857_tv twitch android app](https://user-images.githubusercontent.com/72909307/105608080-e0158280-5db2-11eb-9474-ecc3df023240.jpg)
![Screenshot_20210123_194024_tv twitch android app](https://user-images.githubusercontent.com/72909307/105608081-e1df4600-5db2-11eb-91f6-35983838adb4.jpg)

**Comment by licountrynerd** [comment]
Disable SureStream Adblock in Mod Settings.

**Comment by p3rs0n4l1ty** [comment]
> Disable SureStream Adblock in Mod Settings.

I had it disabled from the beginning

**Comment by licountrynerd** [comment]
Hmm. That's what worked for me when I had the same issue. Not sure then. GL

**Comment by p3rs0n4l1ty** [comment]
Yep. I tried it both enabled and disabled, same thin

**Comment by AlexWayfer** [comment]
I faced with this yesterday. Resolved by re-installation, ad block disabling is not necessary.

**Comment by AlexWayfer** [comment]
Also when I tried to unload the app from RAM I was logged out somewhy (before re-installation).

**Comment by kamild1996** [comment]
Yeah something is scuffed. Sometimes chat will work but streams won't, and it keeps logging me off. Reinstallng the app did not help.

**Comment by p3rs0n4l1ty** [comment]
> I faced with this yesterday. Resolved by re-installation, ad block disabling is not necessary.

Yep, reinstallation helped me but I will keep the issue opened because this bug mist be fixed. Maybe something got corrupted in app cache so cleaning it out may also help in case if it appears again but I have no idea.
